src/main/java/uk/gov/hmcts/reform/sscs/domain/wrapper/AddressDetails.java

```diff
@@ -0,0 +1,88 @@
+package uk.gov.hmcts.reform.sscs.domain.wrapper;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.Objects;
+
+@JsonInclude(value = JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AddressDetails {
```

Comment:
Is this needed? Can't we use the Address object from the common library for consistency?